android/build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.0'
+        classpath 'com.android.tools.build:gradle:7.1.2'
```

Comment:
Out of curiosity, is there any reason why we upgrade to 7.1.2? Does the gradle version here related to the Android version?

Reply:
This bump doesn't impact the android version. Other dependencies appear to have used a higher version than we currently had it set to that was causing SDK XML version conflicts.